### PR TITLE
issue 4 5

### PR DESCRIPTION
**Pull request by abhipsa002**
* [x] Have you followed the guidelines in our
      [Contributing document](https://dvc.org/doc/user-guide/contributing)?

* [ ] Does your PR affect documented changes or does it add new functionality
      that should be documented? If yes, have you created a PR for
      [dvc.org](https://github.com/iterative/dvc.org) documenting it or at
      least opened an issue for it? If so, please add a link to it.

-----

### CODE REVIEW

File: dvc/logger.py [hunk 1]
```diff
@@ -2,14 +2,16 @@
...
 import logging
 import logging.handlers
 import logging.config
 import colorama
 
+from dvc.utils.compat import str, StringIO, RecursionError
+from dvc.progress import Tqdm
+
+
+
```
Comment (efiop) [inline dvc/logger.py]: Please remove these two newlines.

File: dvc/exceptions.py [hunk 2]
```diff
@@ -2,9 +2,11 @@
 
 from __future__ import unicode_literals
 
+import traceback
+
 from dvc.utils.compat import str, builtin_str
 
-import traceback
+
```
Comment (efiop) [inline dvc/exceptions.py]: Please delete this line. No need for three lines.

Comment (abhipsa002) [reply dvc/exceptions.py]: done.

**Comment by efiop** [comment]
Closing due to inactivity. Please feel free to reopen the PR. 🙂Thanks!

**Pull request closed without merging**